lumispy/signals/common_luminescence.py

```diff
@@ -38,42 +39,25 @@
 class CommonLumi:
     """**General luminescence signal class (dimensionless)**"""
 
-    def crop_edges(self, crop_px):
-        """Crop the amount of pixels from the four edges of the scanning
-        region, from out the edges inwards.
-
+    def crop_edges(self, crop_range=None, crop_px=None, rebin_nav=False, **kwargs):
+        """Wrapper around the new crop_edges function.
         Parameters
         ----------
-        crop_px : int
-            Amount of pixels to be cropped on each side individually.
-
-        Returns
-        -------
-        signal_cropped : CommonLuminescence
-            A smaller cropped CL signal object. If inplace is True, the original
-            object is modified and no LumiSpectrum is returned.
+        crop_range : tuple, optional
+            Preferred new parameter. Defines how many pixels to crop per axis.
+        crop_px : int, optional
+            Deprecated. Number of pixels to crop symmetrically.
+        rebin_nav : bool, default False
+            Whether to rebin the navigation axes after cropping.
         """
-
-        width = self.axes_manager.shape[0]
-        height = self.axes_manager.shape[1]
-
-        if crop_px * 2 > width or crop_px * 2 > height:
-            raise ValueError(
-                "The pixels to be cropped cannot be larger than half the width or the length!"
+        if crop_px is not None:
```

Comment:
The warning here is a duplicate of the one in the util function. `crop_px` does not need to appear explicitly in the header, but can be directly handled through `**kwargs` and passed on to the util function. I think then the `if` statement is not even needed and the function becomes a one liner.